pilot/pkg/bootstrap/server.go

```diff
@@ -1424,6 +1442,41 @@ func (s *Server) initEventHandlers() error {
 	return nil
 }
 
+func (s *Server) initSidecarInjector(args *PilotArgs) error {
+	injectPath := args.InjectionOptions.InjectionDirectory
+	if injectPath == "" {
+		log.Infof("Skipping sidecar injector, injection path not specified")
+		return nil
+	}
+	// If the injection path exists, we will set up injection
+	if _, err := os.Stat(injectPath); !os.IsNotExist(err) {
+		dir, err := pilotDnsCertDir()
+		if err != nil {
+			return err
+		}
+		parameters := inject.WebhookParameters{
+			ConfigFile: filepath.Join(injectPath, "config"),
+			ValuesFile: filepath.Join(injectPath, "values"),
+			MeshFile:   args.Mesh.ConfigFile,
+			CertFile:   filepath.Join(dir, "cert-chain.pem"),
+			KeyFile:    filepath.Join(dir, "key.pem"),
+			Port:       args.InjectionOptions.Port,
```

Comment:
How do we enable healthcheck?

Reply:
Why would we need it? We already have pilot health check, the webhook check doesn't do anything especially useful?

Reply:
They are different modules, the healthcheck should cover all, and aggregate all healthchecks

Reply:
Ok good point, will make aggregate check